sagemaker-python-sdk/mxnet_gluon_embedding_server/mxnet_embedding_server.ipynb

```diff
@@ -0,0 +1,591 @@
stchkLiYBTJ8+zsrJCJPXJFbKUShVkuU9nt8n2zgYbax6nTl5idmaSIGmRrxTY2mjjuWW+/eLXePDhIZ7+yCf5zM8+gWbn6bshURLjRyEJUCrajBSnWby0w2J9QGswwEt84kjiN37tCwyPG2QyeXw/pNu58WIyin3slMag71Hb7WJZBv1uj0zOxLIsHCeg1+uRJBGaJhH4N/YWNhsDRoYn6Xbb1HcjpiYK+N6AMAxx3RDXdVFVmSSW8P2QY0c/ge+q3HN8AcNK8Z+/+DXGxmR+73Mf4I//1cvkMhZ3Hj5AYztkenqWQtFET4o89/dn8Lp99PYT/PL/eC9B+w7+r3/3H9k7m8My0/zu7/4+P/czP0HOH+WV106h6Bqf+MCTPPWBR6gUTYoZg34zZHOtw3L1Go32gL33DhHHMZWJItWah22nOX32Cs9+81kiSebi+Sbl4SlsS7m94RMEQRAEQRAEQRAE4aaI0lC45br9HoOuysieHeZn96EmKrJkEHoyqmwQRl1SmSKqptFtO3Q6HY7cPcrilTobaxsUMzmSWKFcNvAGPoEvU13tcuV8k06nRS4zQteLCFoJqRxMTleIkojRsUkKw8NMTZa48+5xIlfipVdOcve9s3h+CydexTTB1A3iOCaMIxzH4/mX3mTbMillNBwSYhwCVIZHC8iSQbfbxXUifD8kTiJkSSGKIgI/wXcTfN/FDwMCv48fdPEdlXbLIY5jwCWIQpyBx9LViKXrmwwNl5DVgPd9KEUcBYyM33hd6DoerhOQzaXx/QCJHIO2waXz15FwUZUhNpZj/sWvv4VLjuYAXnrtNTJljZe/t8jKUotA9SjmdHa2O7xw+VlefaHD0MgKY5kyp5e3qFcD/s8//B2Wvh9gTrcplC02t5q8U3uea71vMZTOMXfYw84NqF+XqF91mJux2bzo09lVOTpznL0H8pipBD9s0W0kFHMmd98zzFuvnaJUlm53/ARBEARBEARBEARBuAmiNBRuOccbMLQn4uTb51hbW0WSJKamZ7n72HHe89gDfOyJf0Jry4UkQdMUGrstTrx4hWLRxNBT7LZXqO40qNerRFEdowiCAAAgAElEQVSALMvIssz4nhGigcn21i5j48MEiouiGywuLrHb2KbebBIFIX3X4bUTb/KTn3mCsfIUv/1bf0htt0693aY/cImJkKQESYLh0XEq5TytTpPLmz3CKCKJZX7pf/g1iuU89UaHODZJEgnXBVmWsWwT00jheSEQ4/kuiqJRLBs/uILsEwQxqirjuTFRGDM1XcKN2oxOGDRqHsVSmsvnBuRSRWTSJEmEoij4vg+JzPDwML7vs3ffY/iBS7vp0mhtsXB4gqERDSuThSDBaxp0dxMi1+fHf/Jx6hs7pHIuz3z9XSrSCJ/9p3eyuujxM5/9aT710SfQFJi/J+H7a88SWREvvvYu+EUmK4c58eoFrmwv0atmMHQd1Q7Ze9d+ZvbcQ+KNsry8zJ8++9ecfb3H4vIuhUyRmVmVJJR47fvXCHwFWRXXkwVBEARBEARBEAThHwJRGgq33A+97yNIkkLo5gjDCM9z2d5dpFgaJ4x2+Zu/+WNKFRtVMWjVO8S+giIb1Hcc0hmTTDHP6OgwuWyFw0fmcZw+Eia6KpPJpShXirhOxJGFw5QyOd7/occYLg3TqrZxuj5RElCulDh35QzV3S3+3R//KidOnEBSNSISoijixldDwg8DiGSajQg7nb7xgjBSWTgyRRxDEum0Gn1cz8N3FdwBBIFHHIc4/RvHTRq1PoEfY5gKuVwOuLHDMY5vlJ1RKBHFPmPj5Ru7GLMSte0Qy7JYOKgycLoUSznq9Sa+7+O6IUkSAzGZ3BSqnKNYypPPJ3QHNZSwhCGH9FptZqYnQNIoDmWo7TYoDWmsXE1AifnV3/xpehs6p991OHnmPO+8eZlTl89w7UKHcqZEqxHwwx87jB43cZ2QBz5wnAPTM+SGbWxGuOfgAUbVOcLGCEppmwiDUjHF0buHMWSdTj0gCBW6/R4HDs+TTpvIkRhPFgRBEARBEARBEIR/CERpKNxyitEmCp0bhVwcYKUlfvhjD3Jt+VWWVmrk8mn6gxZREpFNZ0iZE4SRSzaXZnt9QEmf5dGHnqJSyZCyDXIFk3Qm4ujRo9R2WuzuLhNEHRRTYWe3yuKFNSRJwrAVwsinkC5x/fJ1Fi/77Gz0+OVf+m2yuSGyRQtN05AViElIkoRafRPPC+hqAeNjFcIk5nOf/z263YjdHYedrZBWKyYMFJIkwQ8SJEUllVFJCJFUH9fVgQTPken3HYLAQ1FDTDvB9300Rae2m9Bp+bz1+iLXFzfIl3RGxkv0ez65lImiSOTzeZzBAM/zCAOJYrFMFCU8+OCPE4Q9+ttH0EwD09JYXlwh8Oosr6+SH5IZmsjS9xwGrSEmp0d55KEZTrz7Rb57/lXKJZOvf+kS8/tGMI0ai4ubFMcKvPi1LXr1NqniOHv3zHH2jWvMHipQW28TawO+9fY3eP7Ct3jn+jfZ3N4AX6XbUriycpluO0EyJeJYo1nvsL1dxSp0aDbi2x0/QRAEQRAEQRAEQRBugigNhVuu3nJYWewRBwFmVmVkYohySePQgUMoioJtFjh86G7GplSGRizyIzHTE3tRNJmH7r2DhWMml1ef48r5LVwnQg0NFFlj6dpVDu6d4St//gKf/42/Zm15B12zkRSF3Z0uumYxPDRKEquMjpX5kR99ikceepK245IrjlAo5YjjEM0wCPBJZInsUJnZQ3vIKhYXLl7DDSPePr9OENokmJhpEy/0WN9o4vgOURQRhj69Xpv1tR1s28K2bXRDxrQUfN9F02WSJEFVQZYloigmDCT8YECnnmJifApJimi1d8hkMuSLJr4Xsf/gKLp2Y79hv+8QRxKZdI44qWBbs1Rrr7N2LuHSxfNUt9t4LfjmX3yVrbVt5mam+fhHH8GJ14nCmDe+t81k8DSan6bZWuGnPnsPBBN87MOfYe/8YT
```

Comment:
Please add code to delete the endpoint.

Reply:
Thanks for the review! Find here new commit taking into account modifications https://github.com/awslabs/amazon-sagemaker-examples/pull/882/commits/b54fd5cc668220a6194a00f864733fb67b706657. Edits are the following:

1. Added code to delete the endpoint
2. Moved all the python `import` statement to the top of notebook
3. Simplified the scatter-plot code to contain only needed functionality
4. Removed the local test of the script mode file, which didn't add value
5. Changed deployment instance from `ml.c5.xlarge` to `ml.t2.medium` to minimize charges and make sure that limit-related issues are unlikely to be met by users.

I ran successfully end-to-end in 3 regions (`us-east-1`, `eu-west-1`, `ap-southeast-2`)